mm_cataloguer/generate_manifest.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from modelmeta import DataFile, DataFileVariable, Ensemble, EnsembleDataFileVariables
+from datetime import datetime
+
+def list_filepaths(session, ensemble, since):
```

Comment:
Since *ensemble* is technically a *list* of ensembles, we should use the plural for the variable name.